src/sagemaker/local/image.py

```diff
@@ -98,6 +110,70 @@ def __init__(self, instance_type, instance_count, image, sagemaker_session=None)
         self.container_root = None
         self.container = None
 
+    def process(
+        self, processing_inputs, processing_output_config, environment, processing_job_name
+    ):
+        """Run a processing job locally using docker-compose.
+
+        Args:
+            processing_inputs (dict): The processing input specification.
+            processing_output_config: The processing output configuration specification.
+            environment (dict): The environment collection for the processing job.
+            processing_job_name (str): Name of the local processing job being run.
+
+        Returns:
+        """
```

Comment:
remove this if None? or complete it (with `None`)?

Reply:
Done